Resources/doc/reference/troubleshooting.rst

```diff
@@ -94,6 +94,12 @@ Finally your settings in your ``sonata_media`` parameters will look like this:
                     - sonata.media.provider.file
 
                 formats:
```

Comment:
I'll change this commit as well. You can now omit the `formats` key too. But let's first check the code.